static/js/components/dashboard/CourseDescription_test.js

```diff
@@ -30,6 +30,26 @@ describe('CourseDescription', () => {
     }
   });
 
+  for (let status of ALL_COURSE_STATUSES) {
+    it('shows link view on edX', () => {
```

Comment:
this test name is misleading, it doesn't show the view link for all statuses. Maybe something like 'shows the view on edX link if appropriate' would better capture what you're testing.